### ISSUE TITLE
LDAP plugin not installing on 1.14.2

### ISSUE THREAD

**Issue by unixbird** [osTicket/osTicket#5526]
### Description

I have updated to osticket 1.14.2 and have tried to install the official LDAP plugin and it will not successfully install but will list that the plugin is in the folder

### Versions
Osticket - 1.14.2
Apache  - latest in debian 9
MySQL  - 10.1.44
PHP       - 7.0.33

I have also tested this on this docker image https://hub.docker.com/r/campbellsoftwaresolutions/osticket

and it gives me the same error as I have here. To note both the docker and the debian install of my osticket both have my databases

I am using Debian 9 in prod. 

here is the error that I am getting in the system logs whenever I attempt to install the LDAP plugin

[INSERT INTO ost_plugin SET installed=NOW() ,  install_path='plugins/auth-ldap.phar',  name='LDAP Authentication and Lookup',  isphar=1,  version='0.6.2'] Unknown column 'version' in 'field list'

 ---- Backtrace ----
 #0 (root)/include/mysqli.php(200): osTicket->logDBError('DB Error #1054',  '[INSERT INTO os...')
 #1 (root)/include/class.plugin.php(353): db_query('INSERT INTO ost...')
 #2 (root)/scp/plugins.php(51): PluginManager->install('plugins/auth-ld...')
 #3 {main}

**Comment by JediKev** [comment]
@unixbird 

You will need to add the `version` column to the `ost_plugin` table:
```
ALTER TABLE `ost_plugin`
    ADD COLUMN `version` VARCHAR(64) DEFAULT NULL AFTER `isactive`;
```

We are adding a fix soon, essentially we've never added the `ost_plugin.version` column to any upgrade patches so if you upgraded from pre-1.8 series then you will not have a `version` column (oops).

Cheers.

**Comment by unixbird** [comment]
Rip That'll definitely be the reason. This was a DB that was upgraded from 1.6(iirc). Would I edit this inside the mysql db itself?

**Comment by JediKev** [comment]
@unixbird 

Yes, login to MySQL, use the osTicket database (via `USE db_name;`), then paste/run the query.

Cheers.

**Comment by unixbird** [comment]
I'm still grasping MySQL, is there an ELI5 way of going about this?

**Comment by unixbird** [comment]
@JediKev everything i've done so far with that information is telling me that the syntax is incorrect

**Comment by unixbird** [comment]
Alright scratch all of that, I figured it out

**Comment by JediKev** [comment]
@unixbird 

Not a problem! Glad we go this fixed for you!

Cheers.

**Comment by unixbird** [comment]
I just had to literally copy and paste that one at a time when logging into MySQL. 

1. Login via mysql -u user -p dbname
2. Copy and paste the first line
3. Copy and paste the last line (    ADD COLUMN `version` VARCHAR(64) DEFAULT NULL AFTER `isactive`;) 
4. Then it works

**Comment by JediKev** [comment]
@unixbird 

Just to let you know, I added the `version` column to the below patch:
- https://github.com/osTicket/osTicket/pull/5362

Next upgrade should address this issue for all instances. 👍

Cheers.
